src/pages/AppMarketplace/MultiversxInfographics/index.tsx

```diff
@@ -115,7 +115,7 @@ export const MultiversxInfographics = () => {
 
         const arg = {
           mvxNativeAuthOrigins: [window.location.origin],
-          mvxNativeAuthMaxExpirySeconds: 3000,
+          mvxNativeAuthMaxExpirySeconds: 86400,
```

Comment:
This should stay as 3000. At some point we will switch it to 86400, but when we do it has to be across all apps